src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php

```diff
@@ -69,8 +69,9 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, callable $va
     /**
      * {@inheritdoc}
      */
-    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, callable $index = null, callable $groupBy = null, $attr = null)
+    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, callable $index = null, callable $groupBy = null, $attr = null/*, $labelTranslationParameters = []*/)
```

Comment:
I guess our `DebugClassLoader` is not smart enough yet to detect that the new parameter doc is being added in the parent docblock. So you'll need to add it here to get rid of the current deprecation notice you're facing.

Reply:
Thanks ! I would have never find this !

Reply:
it's not that it's not smart enough - it's that being explicit is needed here, to prove that you cared about adding the argument, event commented

Reply:
It makes sense, thank you for the explanation.